drivers/dma/dma_stm32.c

```diff
@@ -519,9 +516,7 @@ static int dma_stm32_reload(struct device *dev, u32_t id,
 				     size / stream->dst_size);
 	}
 
-	if (IS_ENABLED(CONFIG_DMAMUX_STM32)) {
-		stm32_dma_enable_stream(dma, id);
-	}
+	stm32_dma_enable_stream(dma, id);
 
 	return 0;
 }
```

Comment:
Shouldn't this commit be squashed with the previous? 
Isn't the file the same?

Reply:
Yes, but I did in 2 commits, but yes one change is useless

Reply:
So, did you squash it at the end?